lib/mayaUsd/ufe/ProxyShapeSceneSegmentHandler.h

```diff
@@ -49,6 +49,7 @@ class MAYAUSD_CORE_PUBLIC ProxyShapeSceneSegmentHandler : public Ufe::SceneSegme
     UFE_V4(Ufe::Selection findGatewayItems_(const Ufe::Path& path, Ufe::Rtid nestedRtid)
                const override;)
     bool isGateway_(const Ufe::Path& path) const override;
+    Ufe::Path rootSceneSegmentRootPath() const override;
```

Comment:
This (and method in cpp) need to be ifdef protected using `UFE_PREVIEW_VERSION_NUM`.  Something like:

`#if (UFE_PREVIEW_VERSION_NUM >= 6009)`

Reply:
thank you, see https://github.com/Autodesk/maya-usd/pull/3834/commits/98d81be0a496b5609320560e1682e4de72c4e9ac